Header header
geometry_msgs/Point centroid
geometry_msgs/Point force_bias
geometry_msgs/Point torque_bias